### PR TITLE
adding popularRegions

### PR DESCRIPTION
**Pull request by colins44**

### CODE REVIEW

File: popularRegions/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Popular Regions
+
+Popular Regions are a list of **Destionations** that can be found under a **tag**,
+These Popular Regions currently populate the bottom of the my destinations page.
+
+When a user adds a Popular Region to their alerts they are actually adding all
+the Destinations found within that Popular Regions to the database with the tag_value
+of `popularLocations`
```
Comment (colins44) [inline popularRegions/README.md:8]: Review of this has highlighted that we might not need to be doing this step anymore, this needs to be looked at in more detail.

File: popularRegions/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Popular Regions
+
+Popular Regions are a list of **Destionations** that can be found under a **tag**,
```
Comment (joshuapaling) [inline popularRegions/README.md:3]: Destionations => Destinations

File: popularRegions/tnq.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
...nd":"luxuryescapes","google_result":{"administrative_area_level_1":"Queensland","colloquial_area":"","locality":"Airlie Beach","continent":"Oceania","country":"Australia"},"location_alert":{"geocode":{"lat":-20.2675,"lng":148.716944},"level":"locality","value":"Airlie Beach"}},{"id":"f7b4bd30-6def-47fd-8d25-17759c8dd210","tag_type":"alert","tag_value":"alert","place_id":"ChIJP5xjkRCt2WsR8Iwgf_HuAAU","created_at":"2020-06-04T00:02:02.423Z","brand":"luxuryescapes","google_result":{"administrative_area_level_1":"Queensland","colloquial_area":"Hamilton Island","locality":"Whitsundays","continent":"Oceania","country":"Australia"},"location_alert":{"geocode":{"lat":-20.3512228,"lng":148.9578898},"level":"colloquial_area","value":"Hamilton Island"}},{"id":"8c9188bc-2871-4ab8-96b1-19b3224ec746","tag_type":"alert","tag_value":"alert","place_id":"ChIJEySiW1VieGkRYHggf_HuAAQ","created_at":"2020-06-04T00:00:50.737Z","brand":"luxuryescapes","google_result":{"administrative_area_level_1":"Queensland","colloquial_area":"Cairns","locality":"","continent":"Oceania","country":"Australia"},"location_alert":{"geocode":{"lat":-16.9185514,"lng":145.7780548},"level":"colloquial_area","value":"Cairns"}}]}
```
Comment (joshuapaling) [inline popularRegions/tnq.json:4]: Maybe pretty print this, consistent with other json files above?

**Comment by joshuapaling** [comment]
... and now we've set a default `PORT` env var, probably use that value in CURL requests: `8086`

**Pull request merged**